terraformstate/terraform_state.go

```diff
@@ -19,9 +20,12 @@ type ResourceChange struct {
 	Name          string `json:"name"`
 	ProviderName  string `json:"provider_name"`
 	Change        struct {
-		Actions []string        `json:"actions"`
-		Before  json.RawMessage `json:"before,omitempty"`
-		After   json.RawMessage `json:"after,omitempty"`
+		Actions   []string        `json:"actions"`
+		Before    json.RawMessage `json:"before,omitempty"`
+		After     json.RawMessage `json:"after,omitempty"`
+		Importing struct {
+			Id string `json:"id"`
```

Comment:
@joraff golint is failing as it is Id to be `ID`. Can we change it ?

Reply:
Of course, sorry about that